toptal/picasso
# feat: add timezone property to DatePicker component

[SPC-172]

### Description

- Added optional `timezone` attribute to DatePicker component
- kept the logic to convert to date to timezone within the `DatePicker` component instead of the lower level `Calendar` component because the very same logic had to also be applied to the `DatePicker` input. Thoughts?
- Added some extra tests because these changes touched many parts of the component logic
- Added "WithTimezone" example to Storybook

### How to test

- Open the DatePicker component examples page on Picasso's storybook
- Scroll to `With Timezone` example
- Given a date and an IANA timezone, notice that if the date in the given timezone goes beyond the next or the previous day, it's reflected in the DatePicker input and calendar.

### Review



[SPC-172]: https://toptal-core.atlassian.net/browse/SPC-172
